pkgs/development/node-packages/composition-v4.nix

```diff
@@ -1,4 +1,4 @@
-# This file has been generated by node2nix 1.5.1. Do not edit!
```

Comment:
@ixxie If you had followed my instructions this would not be here. (Just saying ;) )

Reply:
Dang! I did regenerate the files though, so I am not sure what went wrong...

Reply:
Note the only deviation I made from your instructions is not to drop "jupyterhub: various tweaks/fixes" because it included some stuff that was needed for this to work. Then running `./generate.sh` produced both a new `node-packages-v4.nix` and a `node-packages-v6.nix`.

Reply:
@ixxie Your version of the commit with the message 'jupyterhub: various tweaks/fixes' had the commit starting with 8497568a75d while the one I told you to cherry-pick df4b90b was the one made by cstrahan. 

In your version, you have changes to multiple files which are actually not the ones made by cstrahan but happened when you tried to deal with the original rebase.

If you go look at cstrahan's original commit [df4b90b](https://github.com/NixOS/nixpkgs/pull/31950/commits/df4b90b3f43b74eccc0e37c6046bb2bd582e4961) you will notice that he only modified 4 files 
* composition-v4.nix (which is only a newline at end of file.)
* node-packages-v6.json (need to keep)
* node-packages-v6.nix (Which is generated)
* jupyterhub/default.nix (where he uses the node packages)

All of the other file changed in your commit come from the rebase and should not be there.